### DISCUSSION TITLE
Adjusting BMP280 reads?

### DISCUSSION THREAD

**Discussion by aug70** [arendst/Tasmota#17808] [Q&A] [answered]
I have 2 D-1 Minis, with BMP280 attached and I use tasmota sensors on both. They sit next to each other and each shows different values. I have a thermometer that I trust and both BMPs are way off from that thermometer?!? I just want a more accurate read from both devices. Is there a way to adjust temperature in Tasmota sensor? Thank you!

Edit: The sensor is this one -> https://www.amazon.com/GY%E2%80%91BMP280%E2%80%913-3-Atmospheric-Pressure-Electronic-Components/dp/B08R6PBQYP I bought them from AliExpress and they were listed as BMP280.

**Comment by barbudor** [comment, marked as answer]
With no provided values, it's difficult to comment
BMP is normally sold as a pressure sensor with temperature not the main focus. Accuracy is given for +/- 1°C

BME280 on the other hand is advertised as temperature, humidity and pressure and accuracy is given for +/- 0.5°C

AliExpress clones may way out of specs

Also depending how your sensors are mounted relatively to the D1 that is acting as a heater, you can get very bad value unless a few centimeters away

**Reply by sfromis** [reply]
In addition to good distance, also ensure decent airflow around the sensor for ambient temperature

**Comment by aug70** [comment]
Thanks, I started getting better reads when I put a bigger distance between the D1 and sensor
